### DISCUSSION TITLE
Games That Don't Work on Linux and Why Windows Remains Necessary for Gaming

### DISCUSSION THREAD

**Discussion by dillacorn** [dillacorn/win-glaze-dots#1] [General]
### Games with Compatibility Issues
1. **Most Battlefield Games** (EAC)  
2. **Most Call of Duty Games**  
3. **Rust** (EAC Servers)  
4. **Apex Legends** (EAC)  
5. **Fortnite** (EAC)  
6. **Rainbow Six Siege**  
7. **Destiny 2**  
8. **Valorant**  
9. **League of Legends**  
10. **Grand Theft Auto V Online**  
11. **PUBG** (EAC)  
12. **The Crew 2**  
13. **Ring of Elysium**  
14. **Spectre Divide**  
15. **Counter-Strike Nexon**  
16. **Sniper Elite**  
17. **Ace of Spades**  
18. **Roblox** (No longer officially supports Linux)  
19. **KinitoPET** (Made for Windows)  
20. **TinyWhoopGo** (Launcher prevents play even with Proton Experimental)  
21. **Paladins**  
22. **Worlds Adrift**  
23. **Counter-Strike 2 / CS:GO** (ESEA and Faceit platforms)

---

### General Observations
- **Geforce Now**:  
  - Locked to 60 FPS.  
  - Spoofing the client can achieve 120 FPS but only at 720p, which is a poor trade-off.  
  - Requires third-party spoofing tools, which are cumbersome and not worth the effort.  

- **Performance on Windows**:  
  - Frame rates and frame times are generally better.  
  - Example: *The Finals* and *Halo* titles perform better on Windows.

- **ProtonGE**:  
  - Some games require ProtonGE to fix performance or functionality issues.  
  - Example: *Overwatch 2* needs ProtonGE to unlock frame rates beyond 60 FPS.  
  - ProtonGE includes dependencies that Valve cannot legally include in Proton Experimental.  

- **Wallpaper Engine**:  
  - Does not function on Linux.  
  - This may seem obvious but could confuse users unfamiliar with Linux compatibility issues.

- **VR Support**:  
  - Can be challenging to set up depending on the VR hardware.  
  - Some features may be missing or unreliable.  

- **Some Games Don’t Work**:  
  - While rare, there are games that simply will not work on Linux, even with workarounds.  
  - A complete list is hard to compile due to the variety of issues.  

---

### Recommendations
1. **Check Compatibility**:  
   - Always check [ProtonDB](http://protondb.com/) before purchasing a game.  
   - User reports can give valuable insight into compatibility and potential issues.

2. **ProtonDB Limitations**:  
   - ProtonDB only provides a partial view of the experience.  
   - Users may not report graphical bugs or performance issues they encounter later.

**Comment by Usman-butt23** [comment]
Many games don’t work on Linux due to anti-cheat software and lack of native support, making Windows essential for gaming. [Proton-GE](https://protonge.com/) helps run Windows games on Linux, but compatibility issues still exist.

**Comment by flashgnash** [comment]
Red dead online and black desert work I've played both of them on my Linux pc just fine

**Reply by dillacorn** [reply]
I'll remove them from the list then. This is an old post and is def not up-to-date

Linux has improved so much since I made this also
